app/views/users/_user.html.erb

```diff
@@ -0,0 +1,2 @@
+<div id="<%= dom_id user %>">
```

Comment:
This looks strange, but I guess we don't really use this partial?

Reply:
Why not? For admins looking at user's details.

Reply:
I'm confused about the partial being an empty `div`

Reply:
Ah, someone will need to fill something in ;-)